AutoRest/AutoRest.Core/ClientModel/Parameter.cs

```diff
@@ -28,6 +28,20 @@ public Parameter()
         public string Name { get; set; }
 
         /// <summary>
+        /// Gets or sets the parameter client (explicitly defined code generation) name.
+        /// </summary>
+        public string ClientName {
```

Comment:
Do you mean the 'Extensions.cs' file in the Autorest.Generator.Extensions project, or the various ClientModelExtension.cs files among the code generators?